css/dashboards/gui.css

```diff
@@ -416,21 +416,17 @@
 .highcharts-dashboards-edit-context-menu-btn {
     width: 32px;
     height: 32px;
-    margin-left: auto;
+    margin-left: 16px;
+    border-radius: 4px;
     padding: 5px;
-    border-color: var(--highcharts-dashboards-transparent);
-    border-radius: 3px;
+    border: none;
     float: right;
-}
-
-.highcharts-dashboards-edit-context-menu-btn div {
-    height: 4px;
-    background-color: var(--highcharts-dashboards-neutral-color-50);
-    margin: 0 0 4px;
+    cursor: pointer;
 }
 
 .highcharts-dashboards-edit-context-menu-btn:hover {
-    border: 1px solid var(--highcharts-dashboards-neutral-color-80);
+    background-color: var(--highcharts-dashboards-neutral-color-50) !important;
```

Comment:
Please use more accurate selector and avoid using `!important`

Reply:
Yes, no `!important` anywhere, please!